src/util/conversion.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+
+export function boolify(val: string | number | boolean): boolean {
+  const strVal = val.toString().toLowerCase();
+  return strVal !== 'false' && strVal !== '0';
```

Comment:
How about we move these constants into `const FALSY_STRINGS = new Set(['false', '0']);` so we can just do `return FALSY_STRINGS.has(strVal.toString().toLowerCase())`?

This would also make it more extensible.